package.json

```diff
@@ -45,7 +45,7 @@
   },
   "devDependencies": {
     "grunt": "^1.0.1",
-    "grunt-eslint": "^19.0.0"
+    "grunt-eslint": "^18.0.0"
```

Comment:
This commit is the problem:
https://github.com/sindresorhus/grunt-eslint/commit/d0ff94266c668a5d6ad2bfd5bb0e92445f6a580a